src/app/components/calendar.js

```diff
@@ -0,0 +1,78 @@
+import axios from 'axios';
+
+export default function (container) {
+    const active = 'active';
+    const boxes = [...container.querySelectorAll('.js-box')];
+    const boxIcons = [...container.querySelectorAll('.js-icon-box')];
+    const cross = 'js-cross';
+    const hide = 'd-none';
+    const inactive = 'inactive';
+    const buttonTextArray = [...container.querySelectorAll('.js-text-btn')];
+    const mailTimeBtn = container.querySelector('.js-mail-time-submit');
+    const mailTime = container.querySelector('.js-mail-time');
+    const reminder = container.querySelector('.js-reminder');
+    const reminderBtn = container.querySelector('.js-reminder-submit');
+    const resetBtn = container.querySelector('.js-reset-submit');
+    const tick = 'js-tick';
+    const toggleBoxes = [...container.querySelectorAll('.js-toggler')];
+    const toggleTexts = [...container.querySelectorAll('.js-toggler-text')];
+
+    toggleBoxes.forEach((toggler) => {
+        const box = boxes.find(box => box.dataset.dayBox === toggler.dataset.dayBtn);
+        toggler.addEventListener('click', () => {
+            boxToggle(box);
+            postRequest(box.dataset.dayBox, box.classList.contains(active));
+        });
+    });
+
+    reminderBtn.addEventListener('click', () => {
+        const reminderValue = reminder.value;
+        postRequest('reminder', reminderValue);
+    });
+
+    mailTimeBtn.addEventListener('click', () => {
```

Comment:
looks like remind button and mailtime button do almost the same, could you create the universal way. Here you have 2 buttons, imagine you have 1000, will you create 1000 methods?